plugins/performance-lab/includes/site-health/object-cache/helper.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Helper functions used for Object Cache Support Info.
+ *
+ * @package performance-lab
+ * @since 2.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Callback for Object Cache Info fields.
+ *
+ * @return array Fields.
+ * @since 3.3.0
+ */
+function object_cache_supported_fields() {
```

Comment:
Function needs the Performance Lab prefix.

Reply:
fixed